CMakeLists.txt

```diff
@@ -16,6 +16,10 @@ include_directories(SYSTEM ${COMPUTECPP_INCLUDE_DIRECTORY})
 
 enable_testing()
 
+# This will enable ComputeCpp compiler driver if COMPUTECPP_USE_COMPILER_DRIVER
+# was specified and the driver is supported by the current system.
+enable_sycl_compiler_driver()
```

Comment:
I might be missing something here, but I don't think that having to explicitly enable the compiler driver is a particularly good idea, given that it can be enabled/disabled by the user through the cmake cache. Why should someone developing SYCL applications care whether the driver is used or not?

I can just about see a use case, if a developer wishes to have some executables/libraries compiled with the host compiler, then others compiled with the driver, but that would surely require a `disable_sycl_compiler_driver` method too?

I think that this would be better inlined at [when you specify handle the use driver option](https://github.com/codeplaysoftware/computecpp-sdk/pull/111/files#diff-3b92b8ec68465519e2a6b06149381084R223) so that the compiler driver is enabled when the developer calls `find_package(computecpp)`.